config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -527,6 +527,11 @@ spec:
                         type: string
                     type: object
                   type: array
+                messagingTopologyNamespaces:
+                  description: The list of namespaces to allow messaging topology resources to be created from. If one is not defined messaging topology resources can only be created in the same namespace as the cluster.
```

Comment:
I find this to be fairly confusing to those who do not have any background on this. That said, I don't have a much better wording to offer. To me this suggests it may be a difficult feature to explain (or a highly unusual one?)